### PR TITLE
Adding support to HTTPS pattr.me

### PR DESCRIPTION
**Pull request by jeffw16**
pattr.me currently does not redirect HTTPS links to chat.pattr.me (e.g. https://www.pattr.me/c/test will NOT redirect to https://chat.pattr.me/c/test); this should fix that issue.

### CODE REVIEW

**Comment by jeffw16** [comment]
Changes requested IRL have been completed.

File: pattr.py [hunk 1]
```diff
@@ -60,6 +60,8 @@ def generate_nick():
 def enter_chat(roomcode):
     if request.url == 'http://pattr.me/c/' + roomcode or request.url == 'http://www.pattr.me/c/' + roomcode:
         return redirect('http://chat.pattr.me/c/' + roomcode)
+    elif request.url == 'https://www.pattr.me/c/' or request.url == 'https://pattr.me/c/':
```
Comment (justinpotts) [inline pattr.py]: Add in the roomcode to the end of the string to ensure it checks properly.

**Comment by justinpotts** [comment]
Before merging, would it be possible to find a way to force HTTPS?

**Comment by justinpotts** [comment]
(Luke)in good! *Uses the force to merge.*

bb8

**Comment by jeffw16** [comment]
You're killing me with the Star Wars jokes!! XD

**Pull request merged**